module4/collections-ref.py

```diff
@@ -0,0 +1,45 @@
+import random
+import string
+
+
+# Generate list if random number of dicts. Return: List of dicts.
+def generate_random_list_of_dicts(number_of_dicts=10, dicts_values=100):
```

Comment:
It's a good practice to define parameter types in the function. For example: 
def some_function_takes_str_and_returns_list_of_str(some_str: str = "default value") -> list[str]:
    return ["", ""]

Reply:
PEP-484 standard says to use class names and PyCharm triggers on them as I know. I think it is worth to follow this logic. In your example it will be list[dict[str: int]]. Be careful: list[dict{}] is not correct, list[dict[]] is ok.
https://www.python.org/dev/peps/pep-0484/#type-definition-syntax

Reply:
Fix typing by using class names.